### DISCUSSION TITLE
graphql/hive/docs/self-hosting/get-started

### DISCUSSION THREAD

**Discussion by giscus** [graphql-hive/console#5218] [Docs Discussions]
# graphql/hive/docs/self-hosting/get-started

https://the-guild.dev/graphql/hive/docs/self-hosting/get-started

**Comment by hinogi** [comment]
To add some troubleshooting with docker-compose in WSL2
docker-compose tends to throw python errors, to remedy:

`pip3 install --force-reinstall 'requests<2.29.0' 'urllib3<2.0'`
